beacon-chain/sync/validate_beacon_blocks.go

```diff
@@ -125,6 +129,22 @@ func (s *Service) validateBeaconBlockPubSub(ctx context.Context, pid peer.ID, ms
 		return pubsub.ValidationIgnore
 	}
 
+	// Process the block if the clock jitter is less than MAXIMUM_GOSSIP_CLOCK_DISPARITY.
+	// Otherwise queue it for processing in the right slot.
+	currentSlot := s.cfg.Chain.CurrentSlot()
+	if err := helpers.VerifySlotTime(genesisTime, blk.Block().Slot(),
```

Comment:
this is wierd way to check that a block is more than 500 ms, cant we just have an internal helper instead that checks that the block's slot is 500 ms(clock disparity) more than the current time ? It should be easy enough to check here.